app/models/name/create.rb

```diff
@@ -21,50 +26,37 @@ def self.find_names_filling_in_authors(in_str, rank = nil,
   #
   #  names = Name.find_names('Letharia vulpina')
   #
-  def self.find_names(in_str, rank = nil, ignore_deprecated = false,
-                      fill_in_authors = false)
-
+  def self.find_names(in_str, rank = nil, ignore_deprecated: false,
+                      fill_in_authors: false)
     return [] unless parse = parse_name(in_str)
```

Comment:
```suggestion
    return [] unless (parse = parse_name(in_str))
```
This is one of the hundreds of RuboCop offenses that's hidden by `.rubocop_todo.yml`

Reply:
Seems reasonable to fix it if you want.